bandit/core/utils.py

```diff
@@ -319,8 +319,8 @@ def parse_ini_file(f_loc):
         return {k: v for k, v in config.items('bandit')}
 
     except (configparser.Error, KeyError, TypeError):
-        LOG.warning("Unable to parse config file %s or missing [bandit] "
-                    "section", f_loc)
+        LOG.debug("Config file %s not found or missing [bandit] "
```

Comment:
What does the `[bandit]` mean to convey in this debug output? And why is there a trailing space?

Reply:
Ah, sorry! That's saying "... or missing [bandit] section". Got it. (Maybe add "section" at the end?)